Update ANTLR version to 4.9.3

congoamz opened the pull request:
### Description

This commit updates the antlr version used by Babelfish from 4.9.2 to 4.9.3. The 4.9.2 version contains a CMakeLists.txt file that uses the git:// protocol to fetch a particular repository, which will not be supported starting March 15. In the 4.9.3 version the git:// protocol was replaced with the https:// protocol, which works correctly
 
### Issues Resolved

N/A


### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

congoamz committed:
Update ANTLR version to 4.9.3

3manuek commented:
@congoamz @nasbyj this might need to merge new changes as we need this for retagging 1.1.0.

nasbyj committed:
Merge branch 'BABEL_1_X_DEV' into update-antlr

nasbyj committed:
Change ANTLR version in remaining workflows

After merging from BABEL_1_X_DEV we have 3 more workflows. Those need
the same change as for ci.yml.

Signed-off-by: Jim C. Nasby <nasbyj@amazon.com>
